test/metabase/lib/test_util.cljc

```diff
@@ -209,3 +209,42 @@
                                                     :base-type     :type/Integer
                                                     :semantic-type :type/FK}]}
                    :native             "SELECT whatever"}]})
+
+(def categories-mbql-card
```

Comment:
Added several more helper mock Cards & metadata providers with them to facilitate the new tests and to help with other tests in the future.